CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultComplications.uc

```diff
@@ -0,0 +1,304 @@
+
+class X2StrategyElement_DefaultComplications extends X2StrategyElement config(Infiltration);
+
+// Missions that feature lootcrates or other rewards not in its X2RewardTemplates
+var config array<name> LootcrateMissions;
+
+// Items that can have their quantity halved by interception complications
+var config array<name> InterceptableItems;
+
+// Chains that are of great importance to ADVENT and can have Open Provocation complications
+var config array<name> ImportantChains;
+
+// Various values for the complications
+var config int REWARDINTERCEPTION_MIN;
```

Comment:
Please add an underscore (like `REWARD_INTERCEPTION_MIN`) to all of them, it's really hard to read